include/tsv/bits/tsv_misc.hpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2015 Adrien Devresse <adrien.devresse@epfl.ch>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <stdexcept>
+
+#include "../tsv_except.hpp"
+
+namespace TSV {
+
+inline std::vector<TSVInfo> TSVFile::getTSVInfo() {
+    std::vector<TSVInfo> TSVFileInfoVec;
```

Comment:
local variables should be lower cased? Otherwise it looks like a class definition?